### PR TITLE
Feature/testing

### PR DESCRIPTION
**Pull request by HillaryOkello**
In this branch, I wrote the tests for the calculator application.

### CODE REVIEW

**Review by oloomoses — CHANGES_REQUESTED** [review]
## Hello @HillaryOkello,
**You have done a commendable job on testing**
However, there are some issues to be addressed before you can move forward

- [ ] Kindly add unit test for the logic (`calculate.js`). Ensure that all functions are test

## Optional
- [ ] It would be better to update the live version to match the local one

**Please make the above change and ask for another review**

Feel free to leave a comment while mentioning me and I will be happy to reply.
### Happy Coding
_Slack: Oloo Moses_

File: src/logic/__tests__/calculate.test.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  });
+
+  it('Multiplies two numbers', () => {
+    expect(calculate({ next: '1.5*2' }, '+').next).toBe('3+');
+    expect(calculate({ next: '4*7.5' }, '=').next).toBe('30');
+    expect(calculate({ next: '-7*4' }, '-').next).toBe('-28-');
+    expect(calculate({ next: '-8*-4' }, '=').next).toBe('32');
+  });
+
+  it('Divides two numbers', () => {
```
Comment (rindrajosia) [inline src/logic/__tests__/calculate.test.js:18]: - [ ] [REQUIRED] Please add a test when user wants to divide a number by 0 because In your APP, when I do this operation, I got an error:

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Hi, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️

- [ ] It will be nice if you add test units for `all operations` in the calculate.js (eg: 'AC', '%', ...). Don't forget to add negative test units. The aim is to make sure that your APP works correctly

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Please consider to:

- [ ] Update the live demo link.
- [ ] Add scrennshot


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------

**Review by voscarmv — CHANGES_REQUESTED** [review]
# Impressive work.  💻 👀 ✨

## However, there are some changes required 👀 🧰 🔨 🔧
- Please add a test to verify the division by zero functionality, as the previous code reviewer requested.

### Submit these changes and request a new code review! 😄 👍

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier)🧑‍💻🗨️🧑‍💻
# Happy Coding! 🧠 ⌨️ 🖱️ 🖥️

**Review by voscarmv — APPROVED** [review]
# Project status: Approved 💯 ✨ 😎 👍 
### You may merge this branch and move forward ⛙🛣️ 

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻
# Happy Coding! 🧠 ⌨️ 🖱️ 🖥️
